sec-theia-docker/theiahere

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# sec-theia-ide - add a security layer to theia-ide (https and token authentication)
+#
+# https://github.com/dealfonso/sec-theia-ide
+#
+# Copyright (C) GRyCAP - I3M - UPV
+# Developed by Carlos A. caralla@upv.es
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+docker run -u $(id -u) --rm -it -p 10443:10443 -v $(pwd):/home/project calfonso/sec-theia-ide
```

Comment:
I believe this command should be updated.
It will not work in my case or in general.